pkg/operator/ceph/file/mds/livenessprobe_test.go

```diff
@@ -0,0 +1,199 @@
+package mds
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// simple bash wrapper that provides mock behavior for ceph CLI call(s)
+// acts as shim between go test definitions and bash code under test
+// define as string because embed doesn't work in _test.go files
+var probeShimWrapper string = `
+#!/usr/bin/env bash
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+export SCRIPT_DIR
+function ceph() {
+  # IMPORTANT: tests should always define behavior for specific commands being called to ensure code
+  # changes in the future do not inherit invalid success/failure behavior of existing unit tests
+  if [[ "$1" == "fs" ]] && [[ "$2" == "dump" ]] && [[ "$3" == "--mon-host=" ]] && [[ "$4" == "--mon-initial-members=" ]] && [[ "$5" == "--format" ]] && [[ "$6" == "json" ]]; then
```

Comment:
To make sure the probe is rendering and executing correctly, set `somevalue` and `othervalue` to the ROOK_CEPH_MON_... env vars in every test.

```suggestion
  if [[ "$1" == "fs" ]] && [[ "$2" == "dump" ]] && [[ "$3" == "--mon-host='<somevalue>'" ]] && [[ "$4" == "--mon-initial-members='<othervalue>'" ]] && [[ "$5" == "--format" ]] && [[ "$6" == "json" ]]; then
```

Reply:
Not needed i guess, for the test it is working

Should we send env variable as per your suggestion